bigquery/tests/unit/test_table.py

```diff
@@ -70,99 +70,6 @@ def _verifySchema(self, schema, resource):
             self._verify_field(field, r_field)
 
 
-class TestEncryptionConfiguration(unittest.TestCase):
```

Comment:
Can we please keep one test to ensure that `from google.cloud.bigquery.table import EncryptionConfiguration` continues to work? I don't want to make this feature into breaking change.